tests/baselines/reference/objectRestNegative.errors.txt

```diff
@@ -34,8 +33,6 @@ tests/cases/conformance/types/rest/objectRestNegative.ts(17,9): error TS2701: Th
                                     ~
 !!! error TS7008: Member 'y' implicitly has an 'any' type.
         let { x, ...rest } = t;

```

Comment:
This should still be an error until we have higher-order rest types too. At least, I'm pretty sure that `{ y: any }` should actually be `Exclude<T, 'x'>` or something.